webapp-django/crashstats/signature/jinja2/signature/signature_correlations.html

```diff
@@ -0,0 +1,83 @@
+<div id="correlations-wrapper" data-correlations-source="{{ url('crashstats:correlations_json') }}">
```

Comment:
This file is mostly a copy of https://github.com/mozilla/socorro/blob/master/webapp-django/crashstats/crashstats/jinja2/crashstats/partials/correlations.html with a few minor adjustments.